ui/app/templates/components/license-info.hbs

```diff
@@ -71,7 +71,8 @@
     {{#each featuresInfo as |info|}}
       {{#info-table-row label=info.name value=(if info.active "Active" "Not Active") data-test-feature-row="data-test-feature-row"}}
         {{#if info.active}}
-          <ICon @size=28 @glyph="true" /> <span data-test-feature-status>Active</span>
+        <ICon @size=28 @glyph="true" /> <span data-test-feature-status>Active {{#if info.count}}&mdash;
+          count: {{info.count}}{{/if}}</span>
```

Comment:
```suggestion
         {{info.count}} standby nodes connected{{/if}}</span>
```

I think a plain wording might be better here so we're not doing something that looks like "key: value" differently than the rest of the page.

Otherwise I might be interested in a "Details" column and we might have other info about feature usage we can put in?

Reply:
I think it'd be good to add a features column if we add more info around features in the future, but this is the only one that has something other than a string when it's turned on.

What do you think of "standby nodes allowed" ? It's more the number they could connect I don't think the license knows anything about the cluster though.

Reply:
Sure, allowed is better then. Or "Up to {{count}} nodes"